openmmforcefields/generators/template_generators.py

```diff
@@ -766,7 +766,8 @@ def read_file_contents(filename):
             _logger.debug(output)
 
             # Run parmchk.
-            cmd = f"parmchk2 -i out.mol2 -f mol2 -p {self.gaff_dat_filename} -o out.frcmod -s %{self._gaff_major_version}"
+            shutil.copy(self.gaff_dat_filename, os.getcwd())
```

Comment:
It occurs to me that there may also be issues if the *filename* has spaces in it a well. Would it be possible to just use a temporary name `gaff.dat` instead of `os.path.basename(self.gaff_dat_filename)`, so that these would be changed to
```python
            shutil.copy(self.gaff_dat_filename, 'gaff.dat')
            cmd = f"parmchk2 -i out.mol2 -f mol2 -p gaff.dat -o out.frcmod -s %{self._gaff_major_version}"
```

Reply:
Good point. :-) I've made the change as you suggested.